src/PuliServiceProvider.php

```diff
@@ -55,11 +55,17 @@ public function register(Application $app)
     public function boot(Application $app)
     {
         if (isset($app['twig']) && $app['puli.enable_twig']) {
-            $app['twig.loader'] = $app->share(function (Application $app) {
-                return new PuliTemplateLoader($app['puli.repository']);
+            $app->extend('twig.loader', function ($twigLoader, $app) {
+                $twigLoader->addLoader(new PuliTemplateLoader($app['puli.repository']));
```

Comment:
Yes, by default there's a [`Twig_Loader_Chain`](https://github.com/silexphp/Silex/blob/master/src/Silex/Provider/TwigServiceProvider.php#L132-L137), anyway it could be overridden.

I see some possibilities here:

 1. an aggresive strategy, so overriding always the old loader
 2. wrap the current loader with a new Twig_Loader_Chain and than add the Puli loader (:+1:)
 3. don't consider other possibilities, so considering only the Silex default loader

What do you think?